### PR TITLE
Improvements for HeadlineDialog scroll restoration and copy code block button

### PR DESCRIPTION
**Pull request by guanglinn**
Improvements for HeadlineDialog scroll restoration, line numbers in code blocks, copying code block button and other improvements.

### CODE REVIEW

File: app/src/main/java/net/gsantner/markor/activity/DocumentEditAndViewFragment.java [hunk 1]
```diff
@@ -219,6 +218,8 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
         }
         _webView.setBackgroundColor(Color.TRANSPARENT);
 
+        loadDocument();
+
```
Comment (harshad1) [inline app/src/main/java/net/gsantner/markor/activity/DocumentEditAndViewFragment.java:222]: We wan to set the edit text undo redo helper _after_ the first load. Please reorganise this.

**Comment by gsantner** [comment]
has conflicts

**Comment by guanglinn** [comment]
Resolved conflicts :)

**Review by gsantner — APPROVED** [review]
Thank you!

**Pull request merged**